go/tasks/plugins/k8s/spark/spark.go

```diff
@@ -168,6 +168,7 @@ func (sparkResourceHandler) BuildResource(ctx context.Context, taskCtx pluginsCo
 		sparkConfig["spark.kubernetes.executor.limit.cores"] = sparkConfig["spark.executor.cores"]
 	}
 	sparkConfig["spark.kubernetes.executor.podNamePrefix"] = taskCtx.TaskExecutionMetadata().GetTaskExecutionID().GetGeneratedName()
+	sparkConfig["spark.kubernetes.driverEnv.FLYTE_START_TIME"] = strconv.FormatInt(time.Now().UnixNano() / 1000000, 10)
```

Comment:
@EngHabu @anandswaminathan do we have a start timestamp in metadata which can use here ?

Reply:
But is that what he wants to use here? or just time crd was created?

Reply:
@anandswaminathan capturing end to end latency is needed to capture cudtomer feeling, we can add also spark operator separately.

Reply:
@catalinii If that is the case, you need to follow anmol's comment, and try to use started_at of the node. 

Here you are only trying to capture time the crd would be created.

Reply:
Not seeing ExecutableNodeStatus in the plugins and was not able to identify the start time before that. If bring the node startup time into the plugins is too big of a deal we can leave it as it.

Reply:
I believe @anandswaminathan mentioned  started_at in the node_status but that's not passed to the plugin: https://github.com/lyft/flytepropeller/blob/eca2accc887dda4fd9519333d69821f342209d27/pkg/apis/flyteworkflow/v1alpha1/node_status.go#L160 .. @anandswaminathan thoughts ?